python/util.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include "strprintf.hpp"
+
+namespace pyarb {
+namespace util {
```

Comment:
I'm not sure it's wise to have two different namespaces with the same name, despite being properly nested. Won't it lead to surprises?

Reply:
I think that's fine, even common, and otherwise we wouldn't need nested namespaces or even namespaces at all. More confusing would be -- to me -- `pyarb::py_utils` or similar.

Reply:
OK